release-schema.json

```
@@ -30,6 +30,11 @@
           },
           "uniqueItems": true,
           "minItems": 1
+        },
+        "address": {
+          "title": "Address",
+          "description": "An additional contact address. This may be the legally registered address of the organization, or may be a correspondence address for this particular contracting process.",
```

Removing details that are more relevant to organization object.

```suggestion
          "description": "The address of the contact point.",
```